### DISCUSSION TITLE
A pure Csharp Porting of godot-behavior-tree

### DISCUSSION THREAD

**Discussion by MadFlyFish** [kagenash1/godot-behavior-tree#34] [Announcements]
A pure Csharp Porting of godot-behavior-tree. It still needs to be polished but it works now. I will keep up maintaining.
https://github.com/MadFlyFish/godot-behavior-tree-csharp

Thank your for the work your done

![image](https://user-images.githubusercontent.com/36298330/164969945-de38d401-699a-4f6b-8906-de8d8ea5dd4f.png)

**Comment by kagenash1** [comment]
Amazing work! Glad to see the plugin is still useful. I added a link to your version in the readme.
